WordPress/gutenberg
# Data Views: Bulk toolbar covering other clickable elements

## What?
Fixes: #62228
Fixes: #62227 

## Why?
Fixes the `Bulk toolbar` position on scroll. Also fixes the width of it which covers other clickable elements.

## How?
Added relevant CSS

## Testing Instructions
1. Go to site editor
2. Select templates
3. Ensure you have a theme with enough templates to scroll
4. Select one or more items so the bulk toolbar appears
5. Scroll and Scale the browser so the bulk toolbar sits at the same y coordinate as the ellipsis menu for an item in the grid

### Testing Instructions for Keyboard


## Screenshots or screencast 
https://github.com/WordPress/gutenberg/assets/77401999/427801a8-ab2c-47ef-bd6e-257c82910009
